vectordatasource/transform.py

```diff
@@ -9626,15 +9634,30 @@ def capital_alternate_viewpoint(shape, props, fid, zoom):
     return shape, props, fid
 
 
-_ADMIN_LEVEL_TO_KIND = {'2': 'country', '4': 'region', '6': 'county', '8': 'locality'}
+# Map admin level to the kind it should become. Admin_level 3 isn't a widely recognized country,
+# but the only uses of 3 we care about are when it is a country from some viewpoint
+# Similarly 5 is typically used for disputed regions.
+_ADMIN_LEVEL_TO_KIND = {'2': 'country',
+                        '4': 'region',
+                        '6': 'county',
+                        '8': 'locality'}
+
+_PLACE_TO_KIND = {'country': 'country',
+                  'state': 'region',
+                  'region': 'region',
+                  'county': 'county',
+                  'district': 'county',
+                  'locality': 'locality',
+                  'town': 'locality',
```

Comment:
Should this include other place tags? Like:

- province: region
- city: locality
- village: locality

I think we could ignore the other random place tags (hamlet, farm, etc).

Reply:
I'll just include them in the mapping, though they'll probably never be used

Reply:
fixed in 9506b97782500824588c5745c04dc520b202da17